### ISSUE TITLE
Schedule repeat

### ISSUE THREAD

**Issue by craigosborne** [Koed00/django-q#172]
I've just started to look at using Django-q and scheduling tasks. I noticed when testing that the repeat counts down everytime the task runs as per the docs.

However if you have a task that runs every minute it would quickly get to the -32768 limit of a smallint.

Is Django-q not setup for this or is there a better way of doing this?

**Comment by Koed00** [comment]
Good point. When I designed the model , the shortest schedule was 1 hour and I figured it was good enough. The minute option was added much later and I never thought about the repeat field type.

I'd have to change that field to a full integer. Wouldn't be too much of a hassle.

**Comment by craigosborne** [comment]
While I have most of my tasks queued as required it would be useful to be able to run on a schedule when needed.
